configs/components/puppet-ca-bundle.rb

```diff
@@ -19,7 +23,12 @@
     java_available = false
   end
 
-  openssl_cmd = "#{settings[:bindir]}/openssl"
+  if !settings[:vendor_openssl]
+    openssl_cmd = "/usr/bin/openssl"
```

Comment:
This actually came from the cross-compiled linux case, that may have had similar issue I guess, where we want to use the host system openssl instead of using puppetlabs's openssl (which gets used with #{settings[:bindir]}/openssl). Also since now it is limited to el/fedora's this should be ok but can change it to make it platform based if desired.